src/views/product-landing/server.ts

```diff
@@ -9,26 +9,20 @@ import path from 'path'
 import { ProductData } from 'types/products'
 import { stripUndefinedProperties } from 'lib/strip-undefined-props'
 import { validateAgainstSchema } from 'lib/validate-against-schema'
-import { TableOfContentsHeading } from 'layouts/sidebar-sidecar/components/table-of-contents'
-import { BreadcrumbLink } from 'components/breadcrumb-bar'
-import { SidebarProps } from 'components/sidebar'
 import {
 	generateProductLandingSidebarNavData,
 	generateTopLevelSidebarNavData,
 } from 'components/sidebar/helpers'
 import { ProductLandingContent, ProductLandingContentSchema } from './schema'
 import { transformRawContentToProp, extractHeadings } from './helpers'
 import { ProductLandingViewProps } from './types'
+import { SidebarSidecarWithTocProps } from 'layouts/sidebar-sidecar-with-toc'
```

Comment:
🎨 Could this line be moved up to where the previous layout import was?

Reply:
Definitely - updated to move this import up after `lib` imports, and before `component` imports 👍